pkgs/development/compilers/gnu-cobol/default.nix

```
@@ -51,14 +53,27 @@ stdenv.mkDerivation rec {
   # Skips a broken test
   postPatch = ''
     sed -i '/^AT_CHECK.*crud\.cob/i AT_SKIP_IF([true])' tests/testsuite.src/listings.at
+    # upstream reports the following tests as known failures
+    # test 843:
+    sed -i '14180i\AT_SKIP_IF([true])' tests/testsuite.src/run_misc.at
+    # test 875:
+    sed -i '2894s/^/AT_SKIP_IF([true])/' tests/testsuite.src/run_file.at
   '';
 
   preConfigure = ''
     autoconf
     aclocal
     automake
+  '' + lib.optionalString (stdenv.isDarwin) ''
```

```suggestion
  '' + lib.optionalString stdenv.isDarwin ''
```
those are unnecessary. same below.